tests/acceptance/features/webUITextEditor/editTextFiles.feature

```diff
@@ -22,3 +22,27 @@ Feature: textFiles
     And the user closes the text editor
     And the user opens file "New text file.txt" using the webUI
     Then line 1 of the text should be "other text before stuff"
+
+  @issue-core-36233
+  Scenario: Edit restored hidden text file
+    Given user "user1" has uploaded file with content "This is a hidden file" to "/.abc.txt"
+    And the user has enabled the setting to view hidden files on the webUI
+    When user "user1" deletes file "/.abc.txt" using the WebDAV API
+    And user "user1" restores the file with original path "/.abc.txt" using the trashbin API
+    And the user browses to the files page
+    Then as "user1" file "/.abc.txt" should exist
+    When user "user1" downloads file "/.abc.txt" using the WebDAV API
+    Then the downloaded content should be "This is a hidden file"
+#    When the user opens file "/.abc.txt" using the webUI
+#    Then line 1 of the text should be "This is a hidden file"
+
+  Scenario: Edit hidden text file
```

Comment:
Please add `issue-core-36233` tag.
I made this comment in the issue: https://github.com/owncloud/core/issues/36233#issuecomment-623851039
This scenario is also a similar problem - even without delete-restore, a file directly created as a hidden file cannot be opened again with the text editor.

Reply:
Done